srtcore/buffer.cpp

```diff
@@ -1044,9 +1045,9 @@ bool CRcvBuffer::getRcvFirstMsg(ref_t<uint64_t> r_tsbpdtime, ref_t<bool> r_passa
     return false;
 }
 
-bool CRcvBuffer::getRcvReadyMsg(ref_t<uint64_t> tsbpdtime, ref_t<int32_t> curpktseq)
+bool CRcvBuffer::getRcvReadyMsg(steady_clock::time_point &tsbpdtime, ref_t<int32_t> curpktseq)
```

Comment:
Same here, please restore `ref_t`. Removal of `ref_t` is another store, to be done soon.

Reply:
> Same here, please restore ref_t. Removal of ref_t is another story, to be done soon.

I see no reason for not removing `ref_t` if the object is anyway replaced, and the functional logic is preserved.
Furthermore, we don't know the time when this "global removal" will happen.

Reply:
If you really want to fix now, then it should be either:
```
bool CRcvBuffer::getRcvReadyMsg(ref_t<steady_clock::time_point> r_tsbpdtime, ref_t<int32_t> r_curpktseq)
```
or 
```
bool CRcvBuffer::getRcvReadyMsg(steady_clock::time_point& w_tsbpdtime, int32_t& w_curpktseq)
```

But yours matches netither.

Reply:
Changed to 
`bool CRcvBuffer::getRcvReadyMsg(steady_clock::time_point& w_tsbpdtime, int32_t& w_curpktseq)`
Although I don't like this naming.

I would just follow Google's naming convention, that is widely used. It recommends using pointers. Arguments passed by reference should always be const. A function can only modify arguments passed as pointers.
Refer to [Reference Arguments](http://google.github.io/styleguide/cppguide.html#Reference_Arguments) Section.

The same is recommended by Mozilla [style guide](https://firefox-source-docs.mozilla.org/tools/lint/coding-style/coding_style_cpp.html):
> Use pointers, instead of references for function out parameters, even for primitive types.